<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutsideDialog</class>
 <widget class="QDialog" name="OutsideDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Outside</string>
  </property>
  <widget class="QPushButton" name="stepInsideButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Step inside elevator</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>114</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="upButton">
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downButton">
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>158</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Current floor:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="floorLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Door:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>CLOSED</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="floor_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floor_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floor_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floor_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floor_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floor_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>stepInsideButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>floor_1</zorder>
  <zorder>floor_2</zorder>
  <zorder>floor_3</zorder>
  <zorder>floor_4</zorder>
  <zorder>floor_5</zorder>
  <zorder>floor_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
